relay/polkadot/src/lib.rs

```diff
@@ -1970,6 +1940,8 @@ pub mod migrations {
 
 	/// Unreleased migrations. Add new ones here:
 	pub type Unreleased = (
```

Comment:
we need to include a migration for bounties pallets from this PR
https://github.com/paritytech/polkadot-sdk/pull/6255